cmd/git-sync/main.go

```diff
@@ -2111,6 +2144,10 @@ OPTIONS
             The git submodule behavior: one of "recursive", "shallow", or
             "off".  If not specified, this defaults to "recursive".
 
+    --sync-on-signal <string>, $GIT_SYNC_SYNC_ON_SIGNAL
+            Indicate that a sync should occur on receipt of the specifed signal.
```

Comment:
"""
Indicates that an immediate sync should occur upon receipt of the specified signal name (e.g. SIGHUP) or number (e.g. 1).  If not specified, signals will not trigger syncs.
"""